tests/Speckle.Sdk.Tests.Unit/Transports/MemoryTransportTests.cs

```diff
@@ -16,6 +15,6 @@ public sealed class MemoryTransportTests : TransportTests
   [SetUp]
   public void Setup()
   {
-    _memoryTransport = new MemoryTransport(new ConcurrentDictionary<string, string>());
+    _memoryTransport = new MemoryTransport();
```

Comment:
Using a concurrent dictionary is important for this test to not be flaky since we are explicitly testing for concurrent writes.